dataflowengine/src/main/scala/io/shiftleft/dataflowengine/passes/reachingdef/ReachingDefPass.scala

```diff
@@ -277,10 +278,9 @@ class DataFlowFrameworkHelper(graph: ScalaGraph) {
   def getOperation(node: nodes.StoredNode): Option[nodes.StoredNode] = {
     node match {
       case identifier: nodes.Identifier =>
-        if (identifier.argumentIn.hasNext) {
-          getOperation(identifier.argumentIn.nextChecked)
-        } else {
-          None
+        identifier.argumentIn.nextOption match {
+          case None       => None
+          case Some(user) => getOperation(user)
```

Comment:
```suggestion
        identifier.argumentIn.nextOption.map(getOperation)
```

Reply:
Oops, thanks!

Reply:
the suggestion was missing the closing `}`, will remove and merge on green

Reply:
also, you wanted `flatMap` rather than `map` here